pkg/scheduler/framework/plugins/tainttoleration/filtering_test.go

```diff
@@ -171,7 +171,48 @@ func TestFilter(t *testing.T) {
 			wantStatus: nil,
 		},
 		{
-			name: "taints can be tolerated by individual tolerations based on key, effect with Exists operator, every taint has matching toleration - nil status",
+			name: "taints can be tolerated based on key, empty value, effect with Equal operator, every taint has matching toleration - nil status",
+			cluster: &clusterv1beta1.MemberCluster{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-mc",
+				},
+				Spec: clusterv1beta1.MemberClusterSpec{
+					Taints: []clusterv1beta1.Taint{
+						{
+							Key:    "key1",
+							Effect: corev1.TaintEffectNoSchedule,
+						},
+						{
+							Key:    "key2",
+							Effect: corev1.TaintEffectNoSchedule,
+						},
+					},
+				},
+			},
+			policySnapshot: &placementv1beta1.ClusterSchedulingPolicySnapshot{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "csp-1",
+				},
+				Spec: placementv1beta1.SchedulingPolicySnapshotSpec{
+					Policy: &placementv1beta1.PlacementPolicy{
+						PlacementType: placementv1beta1.PickAllPlacementType,
+						Tolerations: []placementv1beta1.Toleration{
+							{
+								Key:      "key1",
+								Operator: corev1.TolerationOpEqual,
```

Comment:
it's not a valid test case? i remember this crp will be rejected by webhook when the value is empty?

Reply:
closed a fix PR to address this https://github.com/Azure/fleet/pull/704 to align with k8s